fixtures/american-dj/mega-tripar-profile.json

```diff
@@ -413,22 +413,22 @@
   "modes": [
     {
       "name": "1-channel",
-      "shortName": "Ch.01",
```

Comment:
Short names should be similar to those displayed on the device's LCD/LED displays. If it is known (e.g. from the manual) to differ from the `1ch` pattern (that most fixtures use), it is fine if fixtures use a different pattern. So please revert all channel `shortName` changes from this PR.

Would you fancy explaining this in the documentation somewhere?